avalon/pipeline.py

```diff
@@ -1198,20 +1191,39 @@ def get_representation_path(representation):
 
     """
 
-    version_, subset, asset, project = io.parenthood(representation)
-    template_publish = project["config"]["template"]["publish"]
-    return template_publish.format(**{
-        "root": registered_root(),
-        "project": project["name"],
-        "asset": asset["name"],
-        "silo": asset["silo"],
-        "subset": subset["name"],
-        "version": version_["name"],
-        "representation": representation["name"],
-        "user": Session.get("AVALON_USER", getpass.getuser()),
-        "app": Session.get("AVALON_APP", ""),
-        "task": Session.get("AVALON_TASK", "")
-    })
+    output = None
```

Comment:
Could you add in the docstring the order of which it resolves the paths in the simplest sense. That way one can easily figure it out without reading the lines of code. It seems to have 3 ways of getting the data and I think it would be good to keep that order and reasoning clear for anyone trying out the function without looking at the code implementation.,

Reply:
This seems to have been addressed.